Resources/public/templates/tabs/locations.hbt

```diff
@@ -73,3 +73,14 @@
         </div>
     </div>
 </div>
+
+<div class="ez-locations-box">
```

Comment:
Not a big fan of the class name. Maybe something like `ez-locations-swap-box` would be a bit better (and adapt below).